api/v1alpha1/nodepool_types.go

```diff
@@ -316,6 +316,52 @@ type NodePoolPlatform struct {
 	Agent *AgentNodePoolPlatform `json:"agent,omitempty"`
 
 	Azure *AzureNodePoolPlatform `json:"azure,omitempty"`
+
+	// PowerVS specifies the configuration used when using IBMCloud PowerVS platform.
+	//
+	// +optional
+	PowerVS *PowerVSNodePoolPlatform `json:"powervs,omitempty"`
+}
+
+// PowerVSNodePoolPlatform specifies the configuration of a NodePool when operating
+// on IBMCloud PowerVS platform.
+type PowerVSNodePoolPlatform struct {
+	// ServiceInstanceID is the ServiceInstance to use for control plane cloud resources.
+	ServiceInstanceID string `json:"serviceInstanceID"`
```

Comment:
which "control plane cloud resources" does this refer to?

Reply:
oops, this should be `NodePool`